docs/c_cxx/README.md

```diff
@@ -1,13 +1,7 @@
-How to build the Doxygen HTML pages on Windows:
+# ONNX Runtime C/C++ docs source files
 
-* Install Doxygen (https://www.doxygen.nl/download.html)
```

Comment:
Ack, I was hoping to keep both sets of instructions since people will likely test things locally vs using the pipeline for everything?

Reply:
I thought they could use the workflow instructions, or do you think that is too much of a stretch?

Reply:
Happy to add these back in if you feel it is necessary.

The other thing is, if you push to your fork, then you can run the GitHub action from there to test